src/runtime/CMakeLists.txt

```diff
@@ -5,6 +5,12 @@ set(HIPSYCL_RT_EXTRA_LINKER_FLAGS "")
 set(HIPSYCL_RT_SANITIZE "" CACHE STRING
     "Enable building hipSYCL runtime with specified sanitizers")
 
+list(PREPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}")
```

Comment:
I'm trying this on the Intel devcloud, because it also has an older toolsystem as a test case. There, cmake is also slightly older and does not yet have `PREPEND`. I guess it's plausible that when users have older toolchains so that they require this PR, they might also have an older cmake. Any reason why we cannot use `APPEND`?

Reply:
We can use `APPEND`, but since we have modified the `FindFilesystem.cmake`, it would be safer to have the path to our version first in the list. `INSERT` can do what we want and works with CMake 3.10 (the one I tested).